tests/python/contrib/test_hexagon/infrastructure.py

```diff
@@ -228,3 +227,18 @@ def compute(n, ho, wo, ko, hi, wi, ki):
         )
 
     return output_shape, compute
+
+
+# Transpose and reshape numpy array according to the specified layout
+def transform_numpy(arr_np, layout):
```

Comment:
The function seems to assume that the supplied layout is NHWC.  Is that a safe assumption for all expected uses of the function?

If no, then should we put `nhwc` into the function name, or perhaps change its argument list to something like `(arr_np, current_layout, new_layout)`?

Reply:
You're right that this function is making an assumption about the supplied layout which can transform the input incorrectly.  Thanks for the suggestion, @cconvey!